boards/DCM/Src/App/states/App_DriveState.c

```diff
@@ -4,35 +4,35 @@
 #include "states/App_AllStates.h"
 #include "states/App_InitState.h"
 #include "App_SetPeriodicCanSignals.h"
+#include "configs/App_MotorEfficiencyLut.h"
 
 #define MAX_TORQUE_REQUEST_NM (90.0f)
 #define EFFICIENCY_ESTIMATE (0.80f)
 #define RPM_TO_RADS(rpm) ((rpm) * (float)M_PI / 30.0f)
-#define IS_EQUAL_FLOAT(a, b) (fabsf((a) - (b)) < 0.005f)
 
 void App_SetPeriodicCanSignals_TorqueRequests(struct DcmCanTxInterface *can_tx, struct DcmCanRxInterface *can_rx)
 {
-    float max_torque_request, torque_request;
-    float bms_available_power   = App_CanRx_BMS_AVAILABLE_POWER_GetSignal_AVAILABLE_POWER(can_rx);
-    float right_motor_speed_rpm = (float)abs(App_CanRx_INVR_MOTOR_POSITION_INFO_GetSignal_D2_MOTOR_SPEED_INVR(can_rx));
-    float left_motor_speed_rpm  = (float)abs(App_CanRx_INVL_MOTOR_POSITION_INFO_GetSignal_D2_MOTOR_SPEED_INVL(can_rx));
-
     // Estimate the maximum torque request to draw the maximum power available from the BMS
     // Note that the motors can not exceed a torque of MAX_TORQUE_REQUEST_NM
-    if (IS_EQUAL_FLOAT(right_motor_speed_rpm + left_motor_speed_rpm, 0.0f))
-    {
-        max_torque_request = MAX_TORQUE_REQUEST_NM;
-    }
-    else
```

Comment:
@timbitss  I didn't quite understand why this was needed. I think a better approach would be to check for zero division.

Reply:
Please suggest how I would check for zero division. Do I use isInfinite(x)?

Reply:
Ah your method works nicely